templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -76,11 +73,9 @@
       </tr>
       <tr class="crm-member-membershiprenew-form-block-renewal_date">
         <td class="label">{$form.renewal_date.label}</td>
-        <td>{$form.renewal_date.html}
-          <div id="defaultNumTerms" class="crm-member-membershiprenew-form-block-default-num_terms description">
-            {ts}Renewal extends Membership Expiration Date by one membership period{/ts}
-            &nbsp;<a id="changeTermsLink" href='#' onclick='changeNumTerms(); return false;'>{ts}change{/ts}</a>
-          </div>
+        <td>{$form.renewal_date.html}<br/>
+          <span class="description">{ts}Renewing will add the normal membership period to the End Date of the previous period for members whose status is Current or Grace and 'Date Renewal Entered' will have no effect.
+<br/>For Expired memberships, renewing will create a membership period commencing from the 'Date Renewal Entered' - if no date is set it will renew from the previous End Date.{/ts}</span>
```

Comment:
Can you split the "ts" into two strings? or remove the "br". For ts to work, it must be all on one line, and ideally no formatting.